helmfile/values/opensearch/securityadmin.yaml.gotmpl

```diff
@@ -0,0 +1,173 @@
+
+  internal_users.yml: |-
+    _meta:
+      type: "internalusers"
+      config_version: 2
+
+    admin:
+      hash: {{ .Values.opensearch.adminHash }}
+      reserved: true
+      opendistro_security_roles:
+      - "all_access"
+      - "manage_snapshots"
+      description: "Admin user"
+
+    dashboards:
+      hash: {{ .Values.opensearch.dashboardsHash }}
+      reserved: true
+      opendistro_security_roles:
+      - "kibana_server"
+      description: "OpenSearch Dashboards user"
+
+    configurer:
+      hash: {{ .Values.opensearch.configurerHash }}
+      reserved: true
+      opendistro_security_roles:
+      - "kibana_user"
+      - "configurer"
+      description: "Configurer user"
+
+  roles.yml: |-
+    _meta:
+      type: "roles"
+      config_version: 2
+
+    # Allows users to view monitors, destinations and alerts
+    alerting_read_access:
+      reserved: false
+      cluster_permissions:
+        - cluster:admin/opendistro/alerting/alerts/get
+        - cluster:admin/opendistro/alerting/destination/get
+        - cluster:admin/opendistro/alerting/monitor/get
+        - cluster:admin/opendistro/alerting/monitor/search
+
+    # Allows users to view and acknowledge alerts
+    alerting_ack_alerts:
+      reserved: false
+      cluster_permissions:
+        - cluster:admin/opendistro/alerting/alerts/*
+
+    # Allows users to use all alerting functionality
+    alerting_full_access:
+      reserved: false
+      cluster_permissions:
+        - cluster_monitor
+        - cluster:admin/opendistro/alerting/*
+      index_permissions:
+        - index_patterns:
+            - kubernetes-*
+            - kubeaudit-*
+          allowed_actions:
+            - indices_monitor
+            - indices:admin/aliases/get
+            - indices:admin/mappings/get
+
+    # Can probably be locked down further
+    configurer:
+      static: false
+      hidden: false
+      reserved: false
+      cluster_permissions:
+      - "cluster:admin/repository/put"
+      - "cluster_manage_index_templates"
+      - "cluster:admin/opendistro/ism/policy/*"
+      index_permissions:
+      - index_patterns:
+        - "*"
+        allowed_actions:
+        - "create_index"
+        - "crud"
+        - "manage_aliases"
+        - "indices:admin/index_template/*"
+        - "indices:admin/opensearch/ism/managedindex"
+        - "indices:admin/rollover"
+        - "indices:monitor/stats"
+
+  # Needed
+  roles_mapping.yml: |-
+    _meta:
+      type: "rolesmapping"
+      config_version: 2
+
+  # Needed
+  tenants.yml: |-
+    _meta:
+      type: "tenants"
+      config_version: 2
+
+  # Needed
+  action_groups.yml: |-
+    _meta:
+      type: "actiongroups"
+      config_version: 2
+
+  # Optional
+  nodes_dn.yml: |-
+    _meta:
+      type: "nodesdn"
+      config_version: 2
+
+  # Optional
+  whitelist.yml: |-
+    _meta:
+      type: "whitelist"
+      config_version: 2
+    config:
+      enabled: false
+
+activeDeadlineSeconds: 1200
+
+resources:
```

Comment:
Maybe expose these values

Reply:
Yeah, I missed to test the resources :sweat_smile:, but now they're exposed and have bit more sane defaults.